cmake/winml_sdk_helpers.cmake

```diff
@@ -16,7 +16,23 @@ function(get_installed_sdk
     set(${sdk_folder} ${win10_sdk_root} PARENT_SCOPE)
 
     # return the sdk version
-    set(${output_sdk_version} ${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION} PARENT_SCOPE)
+    if(CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION)
+        set(${output_sdk_version} ${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION} PARENT_SCOPE)
+    else()
+        # choose the SDK matching the system version, or fallback to the latest
```

Comment:
Ninja doesn't have a target SDK like VS, so if the user doesn't explicitly give one with winml_WINDOWS_SDK_DIR_OVERRIDE / winml_WINDOWS_SDK_VERSION_OVERRIDE I'm picking one by default. I chose to pick one matching the current system version, or fallback to the latest one.

If someone doesn't think this is a sane default and has better suggestions let me know.

Reply:
Just FYI, our CI build machine's system version is less than the target SDK version required  by WinML.